laplacian_variational_autoencoder.py

```diff
@@ -55,11 +55,11 @@ def encode(self, X):
         return Z
 
 
-    def get_loss(self):
+    def get_loss(self, X, Xr):
```

Comment:
(i am assuming you just went through all the autoencoders + changed any call signatures on get_loss + removed references to self.X and self.Xr?)

Reply:
Yup